### DISCUSSION TITLE
unable to register for github education

### DISCUSSION THREAD

**Discussion by NitroH** [community/community#136889] [GitHub Education]
### Hi everyone,

Hello everyone,
I tried many times to subscribe to github education, and everytime i get denied. I was allowed a couple years ago, but not this year. idk why... 
![Screenshot 2024-08-26 220356](https://github.com/user-attachments/assets/0474b122-e5af-40d0-ad0c-65abd5186e90)

I'm sending my school letterhead and as soon as i send it, i recieve an email that says that im denied...
I completed my biling informations well, the picture looks clear, there is the date...
idk what to do in more, to get subscribe knowing that i was already in it last years...
Have a good day, 
I hope you can help me !

**Comment by GabyCT** [comment]
One of the issues that you can have is
[Academic email address already used](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/why-wasnt-my-application-to-github-education-for-students-approved#academic-email-address-already-used)
If your academic email address was already used to request a GitHub Student Developer Pack for a different GitHub account, you cannot reuse the academic email address to successfully apply for another GitHub Student Developer Pack.

Further info https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/why-wasnt-my-application-to-github-education-for-students-approved

**Reply by NitroH** [reply]
Hello, thx for your fast response but i used the same email adress last year for this account. So it cannot be this solution...
